### PR TITLE
Lesson 12

### PR DESCRIPTION
**Pull request by LuciaRider**

### CODE REVIEW

File: TripHouse/main.js [hunk 1]
```diff
@@ -227,57 +227,56 @@ filterRoomsMinus.addEventListener('click', function() {
...
+//       country: 'Germany',
+//       imageUrl: 'https://res.cloudinary.com/intellectfox/image/upload/v1610379364/fe/hostel-friendship_aw6tn7.jpg',
+//     },
+//     {
+//       name: 'Radisson Blu Hotel',
+//       city: 'Kyiv',
+//       country: 'Ukraine',
+//       imageUrl: 'https://res.cloudinary.com/intellectfox/image/upload/v1610379365/fe/radisson-blu-hotel_jwtowg.jpg',
+//     },
+//     {
```
Comment (romos21) [inline TripHouse/main.js]: remove it cause we are working with API now

File: TripHouse/main.js [hunk 2]
```diff
@@ -322,8 +321,14 @@ function createCard(data) {
...
-  swiper.appendSlide(createCard(element))
-});
\ No newline at end of file
+fetch ('https://fe-student-api.herokuapp.com/api/hotels/popular')
+  .then(response => response.json())
+  .then(data => {
+    data.forEach((element) => {
+      swiper.appendSlide(createCard(element))
+    });
+  })
```
Comment (romos21) [inline TripHouse/main.js]: put it into self-invoked function please

File: TripHouse/main.js [hunk 3]
```diff
@@ -321,14 +268,12 @@ function createCard(data) {
...
-      swiper.appendSlide(createCard(element))
-    });
-  })
\ No newline at end of file
+  .then(data => 
+    (function (){
+      data.forEach((element) => {
+        swiper.appendSlide(createCard(element))
+      })
+    }()));
```
Comment (romos21) [inline TripHouse/main.js]: you need to wrap all fetch cause it's asynchronous

**Comment by romos21** [comment]
@LuciaRider having conflicts

**Comment by LuciaRider** [comment]
I accidentally merged branch 12

**Pull request merged**